services/venice-controller/src/main/java/com/linkedin/venice/controller/StoreViewUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.linkedin.venice.controller;
+
+import com.linkedin.venice.controller.kafka.protocol.admin.StoreViewConfigRecord;
+import com.linkedin.venice.meta.ViewConfig;
+import com.linkedin.venice.meta.ViewConfigImpl;
+import com.linkedin.venice.meta.ViewType;
+import com.linkedin.venice.systemstore.schemas.StoreViewConfig;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+public class StoreViewUtils {
+  static Map<String, StoreViewConfigRecord> convertStringMapViewToStoreViewConfigRecord(Map<String, String> stringMap) {
+    // TODO: Today we only support one kind of view and it takes no arguments. So all this returns is a map with a
+    // single
+    // element that is using ChangeCapture as the output type and whatever key is passed in.
+    Map<String, StoreViewConfigRecord> mergedViewConfigs = new HashMap<>();
+    if (!stringMap.isEmpty()) {
+      StoreViewConfigRecord newViewConfigRecord = new StoreViewConfigRecord(ViewType.CHANGE_CAPTURE.value, null);
+      mergedViewConfigs.put(stringMap.entrySet().iterator().next().getKey(), newViewConfigRecord);
+    }
+    return mergedViewConfigs;
```

Comment:
Nit: we might need to differentiate a bit by using `mergedViewConfigRecords` instead of `mergedViewConfigs`? You might copy paste between these two similar methods: `convertStringMapViewToStoreViewConfigRecord` and `convertStringMapViewToStoreViewConfig`( e.g StoreViewConfig newViewConfigRecord at line 32)

Reply:
fixed